### PR TITLE
Server side swift

### PR DESCRIPTION
**Pull request by gbasile**
Microservice build in Swift using Kitura. 

- Implement canned responses for the `/microservices` ENDPOINT
- Swagger documentation for the entrypoint

Next: 
- Testing
- Deployment
- Authentication
- Persistance

### CODE REVIEW

File: microservices-swift/.bluemix/toolchain.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+version: '2'
```
Comment (florianmski) [inline microservices-swift/.bluemix/toolchain.yml]: We can probably trash the entire `.bluemix` folder

File: microservices-swift/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!groovy
```
Comment (florianmski) [inline microservices-swift/Jenkinsfile]: Probably don't need this

File: microservices-swift/README.rtf [hunk 3]
```diff
@@ -0,0 +1,184 @@
+{\rtf1\ansi\deff0{\fonttbl{\f0 \fswiss Helvetica;}{\f1 Courier;}}
```
Comment (florianmski) [inline microservices-swift/README.rtf]: Trash this as well

File: microservices-swift/chart/microservices-swift/values.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# This is a YAML-formatted file.
```
Comment (florianmski) [inline microservices-swift/chart/microservices-swift/values.yaml]: Can probably also trash everything under /chart

File: microservices-swift/terraform/variables.tf [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# Default Data Center for the creation of the Virtual Instance
```
Comment (florianmski) [inline microservices-swift/terraform/variables.tf]: Can also probably trash everything under /terraform

File: microservices-swift/.bluemix/toolchain.yml (hunk 1 above)
Comment (gbasile) [reply microservices-swift/.bluemix/toolchain.yml]: Done here 6116418f68ed172e1a7fa82a96d21ea4c745c4a4

File: microservices-swift/terraform/variables.tf (hunk 5 above)
Comment (gbasile) [reply microservices-swift/terraform/variables.tf]: f83a03b414306be44258998ad446b86d505d0cfd

File: microservices-swift/README.rtf (hunk 3 above)
Comment (gbasile) [reply microservices-swift/README.rtf]: f83a03b414306be44258998ad446b86d505d0cfd

File: microservices-swift/Jenkinsfile (hunk 2 above)
Comment (gbasile) [reply microservices-swift/Jenkinsfile]: f83a03b414306be44258998ad446b86d505d0cfd

File: microservices-swift/iterative-dev.sh [hunk 6]
```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# This script sets up an iterative loop to accelerate development of
```
Comment (xrigau) [inline microservices-swift/iterative-dev.sh]: this is cool!

Comment (gbasile) [reply microservices-swift/iterative-dev.sh]: yep, it works like a charm!

**Pull request merged**